### ISSUE TITLE
Unable to enable traces when connecting with Effectiveusername

### ISSUE THREAD

**Issue by helgews** [DaxStudio/DaxStudio#539]
Since version 2.14 I can no longer enabler Server Timings trace when connecting as admin with effectiveusername "testuserXX". 
This is confirmed to be working up until 2.13.1.

1. Connect to my Azure Analysis Services (1500) with DAX Studio 2.14.0 / 2.14.1 / 2.14.2 preview
    - pass effectiveusername also, to test the RLS for a given user
    - Wait for credentials selection in a small browser window, select my own account (AS admin)
    - Connection ok, metadata loaded
    - try to enable server timings trace, and get this error

![image](https://user-images.githubusercontent.com/56156227/106713026-1e544280-65fa-11eb-806b-774f82e9bad1.png)
Complete error message in english: "Error enabling Trace: The user [...] does not have permission to call the Discover-method. Technical details: [...]

I am happy to provide additional details if nescessary!

Thank you for this great product!

**Comment by dgosbell** [comment]
Thanks for logging this. It's actually a side effect of the new xmSQL simplification feature we added in 2.14.0. We have to use a Discover call that requires admin rights to get some of the information we require for this, but it is not available when using parameters like Roles or EffectiveUserName that alter the permissions of the connection.

I was able to figure out a way of skipping the offending code and displaying a warning, but I'm now testing a new approach that lets us open a second temporary connection without the Roles/EffectiveUserName which lets the xmSQL simplification features continue to work.
